core/templates/services/UpgradedServices.ts

```diff
@@ -389,6 +389,9 @@ import { StateClassifierMappingService } from
 import { StateContentService } from
   // eslint-disable-next-line max-len
   'components/state-editor/state-editor-properties-services/state-content.service';
+import { StateLinkedSkillIdService } from
```

Comment:
Changes to this file are not required anymore.

Reply:
Done.